src/bfx_tools/isovar.ml

```diff
@@ -0,0 +1,66 @@
+open Biokepi_run_environment
+open Common
+
+module Configuration = struct
+
+  type t = {
+    name: string;
+    min_reads: int;
```

Comment:
FYI: I'm going to change this parameter (or add another) when I add support for counting partially overlapped variant reads.